src/presentation/pages/Home.vue

```diff
@@ -8,15 +8,18 @@
     >
       CodeX UI showcase
     </Heading>
-    <Button text="Button text" />
+    <div :class="$style.button">Test</div>
+    <Button
+      text="Button text"
+    />
     <Input text="Input text" />
   </div>
 </template>
 
 <script setup lang="ts">
 import { useHead } from 'unhead';
 import { useI18n } from 'vue-i18n';
-import { Button, Input, Heading } from 'codex-ui/vue';
+import { Button, Input, Heading } from '../../../codex-ui/src/vue';
```

Comment:
it should be used from workspace, like it was done before

Reply:
![image](https://github.com/codex-team/notes.web/assets/99203910/bee82c1e-18bf-40b6-9c1e-452ebec43002)

Because I was receiving error

Reply:
If `yarn build` is working, that means that there is a problem with VSCode. TS config aliases used to let VScode understand that import. I'm not sure why it is not working in your case, need to research it.

Reply:
make sure you've run `yarn build` in `codex-ui`